src/libasr/pass/instantiate_template.cpp

```diff
@@ -250,9 +283,10 @@ class FunctionInstantiator : public ASR::BaseExprStmtDuplicator<FunctionInstanti
                 int64_t result;
                 switch (op) {
                     case (ASR::binopType::Add): { result = left_value + right_value; break; }
-                    case (ASR::binopType::Sub): { result = left_value - right_value; break; }
-                    case (ASR::binopType::Mul): { result = left_value * right_value; break; }
+                    // case (ASR::binopType::Sub): { result = left_value - right_value; break; }
+                    // case (ASR::binopType::Mul): { result = left_value * right_value; break; }
```

Comment:
Why is this commented out?

Reply:
I haven't made any tests with subtraction and multiplication and there is no restriction defined that corresponds to these operations yet.